### DISCUSSION TITLE
Create new release of gitweb?

### DISCUSSION THREAD

**Discussion by manojkarthick** [yoannfleurydev/gitweb#74] [Q&A] [answered]
Thanks for building this awesome tool, it's been really useful! 💯 

I'm trying to build the project using the latest release v0.3.1 via [macports](https://www.macports.org/), but the build fails due to a disallowed version of anyhow. I see that the anyhow dependency has been [updated](https://github.com/yoannfleurydev/gitweb/commit/19201d63370c47ece9742e4bc3d7cb9127eed19f) in the master branch by dependabot, so master seems to build properly. 

Would it be possible to create a new patch version v0.3.2 from master that contains the updates? Thanks!

**Comment by yoannfleurydev** [comment, marked as answer]
Hi @manojkarthick , thanks a lot for you feedback 😊 

I didn't know that the version of `anyhow` caused building problems, thanks a lot for letting me know.

I just created the tag and pushed it, the `v0.3.2` is currently releasing itself. 🚀

**Reply by manojkarthick** [reply]
Thanks so much for the quick feedback! 🚀
